infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutContextData.kt

```diff
@@ -34,6 +34,30 @@ interface LayoutContextAware<T> {
     val isDesign: Boolean get() = false
 }
 
+sealed class ContextIdentity<T> {
```

Comment:
Tässä on tuo reviewin-keskustelun varsinainen tuotos, eli aiemmin kontekstilla oli erilliset kentät id, rowid, version joista vähän kaikkien piti olla nullablea. Oikeasti kaikki combot noista ei ole mielekkäitä vaan vaihtoehtoja on 3 (alla). Näiden luokkien nimet on minusta vähän huonot, joten jos joku osaa ehdottaa fiksumpia niin otetaan mieluusti vastaan.

Reply:
Munkin mielestäni nämä on fiksua erottaa omiksi tyypeikseen 👍 Jotenkin tosin tuo `Identity`-termi kinnaa mulla vastakarvaan, varmaan kun meillä on jo käytetty niin paljon `Id`-termiä muualla, ja tämä on terminä hyvin sinnepäin, mutta kuitenkin lopulta aika eri asia. Toisaalta kuvaava se kyllä on... Enpäs oikein osaa kyllä antaa parannusehdotuksiakaan 😞

Reply:
Tämän ja alemman huomioiden nimesin nämä nyt:
```
sealed class ContextIdHolder<T>
data class UnstoredContextIdHolder<T>
data class StoredContextIdHolder<T>
data class OverwritingContextIdHolder<T>
```